cluster-stamp.bicep

```diff
@@ -1871,10 +1919,10 @@ resource acrKubeletAcrPullRole_roleAssignment 'Microsoft.Authorization/roleAssig
   }
 }
 
-// Grant the OMS Agent's Managed Identity the metrics publisher role to push alerts
-resource mcOmsAgentMonitoringMetricsPublisherRole_roleAssignment 'Microsoft.Authorization/roleAssignments@2020-10-01-preview' = {
+// Grant the Azure Monitor (fka as OMS) Agent's Managed Identity the metrics publisher role to push alerts
```

Comment:
really good to know.

Reply:
Yeah, they just released a blog post on the name change, even the pods changed name in the cluster.  Branding issues with OMS